dotcom-rendering/playwright/lib/mocks.ts

```diff
@@ -0,0 +1,22 @@
+import type { Page } from '@playwright/test';
+import { mostRead } from '../fixtures/manual/most-read';
+import { mostReadGeo } from '../fixtures/manual/most-read-geo';
+
+export const mockApis = (page: Page): Promise<[void, void, void]> => {
+	const mostReadPromise = page.route('**/most-read/**', (route) => {
+		return route.fulfill({
```

Comment:
`route.fulfill` is how we override a response:
https://playwright.dev/docs/api/class-route#route-fulfill